packages/eslint-config-javascript/rules/imports.js

```diff
@@ -1,15 +1,4 @@
-module.exports = {
-    env: {
-        es6: true
-    },
-    parserOptions: {
-        ecmaVersion: 6,
-        sourceType: 'module'
-    },
-    plugins: [
-        'import'
-    ],
-
+export const imports = {
```

Comment:
Can you comment on the deletion of this configuration? Is there no longer a way to configure these settings? Does this result in a behavior change?

```js
    env: {
        es6: true
    },
    parserOptions: {
        ecmaVersion: 6,
        sourceType: 'module'
    },
    plugins: [
        'import'
    ],
```

Reply:
This applies to `rules/es6.js` as well.